### PR TITLE
Fixes posterpage video not properly functioning

### PR DESCRIPTION
**Pull request by mike-lvov**
There is a problem when u try to access the poster directly and there is a person already broadcasting, you won't hear him/her

### CODE REVIEW

File: src/components/templates/PosterPage/PosterPage.tsx [hunk 1]
```diff
@@ -79,6 +84,10 @@ export const PosterPage: React.FC<PosterPageProps> = ({ venue }) => {
   const hasFreeSpace =
     videoParticipants.length < POSTERPAGE_MAX_VIDEO_PARTICIPANTS;
 
+  if (!hasRoom || !hasParticipants || isRoomLoading) {
+    return <Loading />;
```
Comment (goran-peoski-work) [inline src/components/templates/PosterPage/PosterPage.tsx:88]: Could this `!hasRoom || !hasParticipants` make it so it looks like endless loading? Could "not found" message be more appropriate?

**Review by colinhowe — APPROVED** [review]

File: src/hooks/twilio/useParticipantMediaState.ts [hunk 2]
```diff
@@ -53,6 +53,8 @@ export const useParticipantMediaState = (
     },
   ] = useList<VideoTrack | AudioTrack>();
 
+  console.log(media, tracks);
```
Comment (goran-peoski-work) [inline src/hooks/twilio/useParticipantMediaState.ts]:
```suggestion
```

File: src/hooks/twilio/useVideoRoomState.tsx [hunk 3]
```diff
@@ -23,14 +23,25 @@ export const useVideoRoomState = (
   activeParticipantByDefault = true
 ) => {
   const [room, setRoom] = useState<Room>();
+
   const [videoError, setVideoError] = useState("");
   const dismissVideoError = useCallback(() => setVideoError(""), []);
+
+  const hasRoom = room !== undefined;
```
Comment (goran-peoski-work) [inline src/hooks/twilio/useVideoRoomState.tsx:30]: what about `null`? could `isDefined()` be used?

**Review by goran-peoski-work — APPROVED** [review]
Looks OK, few questions posted, if not important could be merged I guess.

File: src/components/templates/PosterPage/PosterPage.tsx (hunk 1 above)
Comment (mike-lvov) [reply src/components/templates/PosterPage/PosterPage.tsx:85]: Not really possible, but we gain nothing from these checks either way. So I removed them

File: src/hooks/twilio/useParticipantMediaState.ts (hunk 2 above)
Comment (mike-lvov) [reply src/hooks/twilio/useParticipantMediaState.ts]: Removed

**Comment by mike-lvov** [comment]
Is not relevant anymore

**Pull request closed without merging**